privacyidea/lib/policy.py

```diff
@@ -2278,6 +2279,10 @@ def get_static_policy_definitions(scope=None):
                           'allow authentication with the same credentials for a '
                           'certain amount of time. '
                           'Specify timeout like 4h or 4h/5m.')
+            },
+            ACTION.PREFERREDCLIENTMODE: {
+                'type': 'str',
+                'desc': _('preferred client mode')
```

Comment:
It would be nice to add possioble values here:

~~~Python
   'value' : [...] 
~~~~

and take them from the client mode definition.

Ah, oh wait. We said we could add a list of preferred values.

So please enhace the `'desc'` and explain, which modes are available. Compare the action `AUTH_CACHE` above. There should be more hints!

Reply:
[34ee918](https://github.com/privacyidea/privacyidea/pull/3361/commits/34ee918173549cdef4dfeb1e000951457fff1cab)